content/about/faq.md

```diff
@@ -1,23 +1,54 @@
 
-From Google:
-> *quire: 1. four sheets of paper or parchment folded to form eight leaves, as in medieval manuscripts. 2. any collection of leaves one within another in a manuscript or book. 3. 25 (formerly 24) sheets of paper; one-twentieth of a ream.*
+### What are the benefits of 11ty?
 
-In Latin:
-> *Quire is also a form of the Latin verb “queo”, or “I am able” or “I can”. Specifically, it’s the 2nd principal part and so means “to be able”. It’s also intransitive and expects an action. To be able to write, “quire scribere”. To be able to publish, “quire prodere”. To be able to edit, “quire corrigere”. Quire literally “can” do anything.*
+11ty has a significantly faster processing time than other static site generators. With the move to 11ty, we have reworked the overall file structure of Quire projects to simplify navigation and make essential project files more accessible. On the development side, one of 11ty's great strengths is its ability to mix templating languages. For example, in Quire projects, Liquid will be used to generate simple markup while JavaScript will be used where data manipulation is required (more on those two langauges below.) Other templating languages may also be used. Above all, the move to 11ty will enable the expansion of advanced features, a more robust set of CLI options, and improve developer support and contribution opportunities.
 
-**Who makes up the Quire community?**
+### What is JavaScript?
 
-**Why Open Source?**
+Javascript is a popular text-based programming language that turns static web pages into dynamic, interactive sites, going a step beyond the capabilities of HTML and CSS. JavaScript can be found everywhere on the web, from refreshing a Twitter feed to your Amazon search bar or embedding Youtube videos. It is also widely used in the art, cultural heritage, and academic sectors. Learn more about JavaScript on the [Mozilla developer website](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript).
 
-**How can I participate?**
+### What is Liquid?
+
+Liquid is an open-source templating language written in [Ruby](https://www.ruby-lang.org/en/). It was created by the e-commerce company [Shopify](https://www.shopify.com/) in 2006. It is the backbone of all Shopify themes and is used to load dynamic content on website storefronts. Read more about Liquid on the [Shopify.dev website](https://shopify.dev/api/liquid).
+
+### What is Vite
+
+[Vite (Vite.js)](https://vitejs.dev/) is a lightning fast tool for frontend development. It is replacing [Webpack](https://webpack.js.org/) and will be used to compile styles and scripts as well as bundle files.  
```

Comment:
"lightning fast tool for frontend development" sounds copy-pasted straight from their website. I'm not 100% clear on it myself though, maybe worth asking the team?